dev_guide/application_ephemeral_storage_sizing.adoc

```diff
@@ -0,0 +1,143 @@
+    be greater than or equal to the ephemeral storage request.
+
+* *Administration*
+
+  - The cluster administrator may assign quota against the ephemeral storage request value,
+    limit value, both, or neither.
+
+  - The cluster administrator may assign default values for the ephemeral storage request
+    value, limit value, both, or neither.
+
+  - The cluster administrator may override the ephemeral storage request values that a
+    developer specifies, in order to manage cluster overcommit. This occurs on
+    OpenShift Online, for example.
+
+== Strategy
+
+The steps for sizing application ephemeral storage on {product-title} are as follows:
+
+. *Determine expected container ephemeral usage*
++ If your administrator has enabled the ephemeral-storage tech preview in {product-title} 3.10, 
+determine expected mean and peak container ephemeral storage usage, empirically if
+necessary (for example, by separate extended). Remember to consider all the
+processes that may potentially run in parallel in the container: for example,
+does the main application spawn any ancillary scripts that may require local storage for work files or logging?
+
+. *Determine risk appetite*
++
+Determine risk appetite for eviction. If the risk appetite is low, the
+container should request ephemeral storage according to the expected peak usage plus a
+percentage safety margin. If the risk appetite is higher, it may be more
+appropriate to request ephemeral storage according to the expected mean usage.
+
+. *Set container ephemeral storage request*
++
+Set container ephemeral storage request based on the above. The more accurately the
+request represents the application ephemeral storage usage, the better. If the request is
+too high, cluster and quota usage will be inefficient. If the request is too
+low, the chances of application eviction increase.
+
+. *Set container ephemeral storage limits, if required*
++
+Set container ephemeral storage limits, if required. Setting a limit has the effect of
+immediately killing a container process if the combined ephemeral storage usage of all
+processes in the container exceeds the limit, and is therefore a mixed blessing.
+On the one hand, it may make unanticipated excess ephemeral storage usage obvious early
+("fail fast"); on the other hand it also terminates processes abruptly.
++
+Note that some {product-title} clusters may require a limit value to be set;
+some may override the request based on the limit; and some application images
+rely on a limit value being set as this is easier to detect than a request
+value.
++
+If these limits are set, they should not be set to less than the expected peak
+container resource usage plus a percentage safety margin.
+
+. *Ensure application is tuned*
++
+Ensure application is tuned with respect to configured request and limit values,
+if appropriate. This step is particularly relevant to applications which pool
+ephemeral storage.
+
+== Diagnosing an Evicted Pod
```

Comment:
"Diagnose" what? What's this section really about? What makes exhaustion of ephemeral storage happen?
Section seems to discuss that what can happen to a pod when ephemeral storage is used up, maybe? How about explaining what exhausts this storage and how users can avoid doing that?

Reply:
It is about what happens when the storage is used up.

Again, please see `architecture/additional_concepts/ephemeral-storage.adoc` -- that's intended to be the overall explanation of it.

Reply:
Resolved.